spec/unit/puppet/provider/pam/augeas_spec.rb

```diff
@@ -131,6 +131,31 @@
                           :arguments => ["try_first_pass","retry=3","type="],})
     end
 
+    describe "when reodering settings" do
+      it "Changing the order of an entry" do
+        aug_open(target, "Pam.lns") do |aug|
+          expect(aug.get("./2/module")).to eq("pam_unix.so")
```

Comment:
I don't think it's necessary to test the initial state, we should know it already.